setup.cfg

```
@@ -114,7 +114,7 @@ ja =
     sudachipy>=0.5.2,!=0.6.1
     sudachidict_core>=20211220
 ko =
-    natto-py==0.9.0
+    natto-py>=1.0.0,<2.0.0
```

I don't think there's any particular need to forbid the older versions, since most people will get the most recent one when they install anyway.

```suggestion
    natto-py>=0.9.0
```

None of this is enforced or checked except on the initial install and you can override it without formal conflicts, so open ranges are less of a concern here.